e2e-tests/portal/cypress/e2e/journeys/maker/dashboard/deals/dashboard-table-data.spec.js

```diff
@@ -65,9 +65,8 @@ context('View dashboard deals as a maker', () => {
 
   it('BSS and GEF deals render on the dashboard with correct values', () => {
     cy.login(BANK1_MAKER1);
-    dashboardDeals.visit();
 
-    const { exporter, bankRef, product, status, type, updated } = dashboardDeals.row;
+    const { updated } = dashboardDeals.rowIndex;
```

Comment:
This is really odd - I dont think `updated` makes sense on its own

Reply:
This hold the information of deals on dashboard page

Reply:
it's a cypress selector, I agree with @MarRobSoftwire `updated` is not clear here, e.g `rowIndex.updated()` or `dashboardDeals.rowIndex.updated()` would be clearer here

Reply:
To be checked.